src/Microsoft.AspNetCore.Sockets.Client/Connection.cs

```diff
@@ -21,6 +21,7 @@ public class Connection: IConnection
         private ITransport _transport;
         private Task _receiveLoopTask;
         private volatile Task _startTask = Task.CompletedTask;
+        private volatile Task _eventQueue = Task.CompletedTask;
```

Comment:
I thought we eventually did not move it. Since we did, I will change the code to use it.